src/journal.c

```diff
@@ -712,13 +712,13 @@ on_close_clicked_cb (GtkWidget *w, gpointer data)
 	 * pop after the destroy has happened. */
 	if (wig->hover_timeout_id)
 	{
-		gtk_timeout_remove (wig->hover_timeout_id);
+		g_source_remove (wig->hover_timeout_id);
```

Comment:
In this file the return types of `hover_kill_func` and `hover_timer_func` could be adjusted too.